posthog/tasks/exports/image_exporter.py

```diff
@@ -238,6 +241,15 @@ def _screenshot_asset(
         """
         )
 
+        if max_height_pixels and height > max_height_pixels:
```

Comment:
The caveat with this approach is that chrome still has to load the entire table, but at least it doesn't have to screenshot the entire table.